editor/project_converter_3_to_4.cpp

```diff
@@ -465,6 +465,7 @@ static const char *gdscript_function_renames[][2] = {
 	{ "post_import", "_post_import" }, // EditorScenePostImport
 	{ "print_stray_nodes", "print_orphan_nodes" }, // Node
 	{ "property_list_changed_notify", "notify_property_list_changed" }, // Object
+	{ "raise","move_to_front" }, // CanvasItem
```

Comment:
```suggestion
	{ "raise", "move_to_front" }, // CanvasItem
```
To pass CI

Reply:
Please do that change by *amending* the commit with `git commit --amend`, and then `git push --force` to update the PR, so it stays as a single commit.
See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request) for instructions.

Reply:
Commit amended, thank you all!